src/client/Client.cc

```diff
@@ -6182,7 +6182,7 @@ void Client::tick()
   ldout(cct, 21) << "tick" << dendl;
   tick_event = timer.add_event_after(
     cct->_conf->client_tick_interval,
-    new FunctionContext([this](int) {
+    new LambdaContext([this](int) {
```

Comment:
my only concern is that this will increase the binary size. as each instantiate of the `LambdaContext` will create a new instance, IIUC.